test/support/test_helpers.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Skillswheel.TestHelpers do
+  alias Skillswheel.Repo
+
+  def insert_user(attrs \\ %{}) do
+    changes = Dict.merge(%{
+      name: "Some User",
+      username: "user #{Base.encode16(:crypto.rand_bytes(7))}",
```

Comment:
we don't have `username` in our users table. We can look at adding this if necessary but email, name and password should be fine for now